python/sglang/srt/layers/moe/fused_moe_triton/fused_moe.py

```diff
@@ -268,7 +270,7 @@ def moe_align_block_size(
     )
     num_tokens_post_pad = torch.empty((1), dtype=torch.int32, device=topk_ids.device)
     # FIXME(zhyncs)
-    if num_experts >= 256:
+    if num_experts >= 256 and not is_hip():
```

Comment:
always cache the result of `is_hip` instead of calling it every time during the runtime